src/pages/XvmTransfer.vue

```diff
@@ -8,11 +8,20 @@ import { defineComponent } from 'vue';
 import { useMeta } from 'quasar';
 import XvmTransfer from 'src/components/assets/transfer/XvmTransfer.vue';
 import { usePageReady } from 'src/hooks';
+import { meta } from 'src/config/metadata';
 
 export default defineComponent({
   components: { XvmTransfer },
   setup() {
-    useMeta({ title: 'XVM Transfer' });
+    useMeta({
```

Comment:
Maybe to define a function which accepts title and returns metadata object. I see very similar code on lot of places.

Reply:
@bobo-k2  Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/689/commits/cc70f25e2c0f95d6c4778730d193097a551c256a) commit.